fvm/src/engine/mod.rs

```diff
@@ -158,16 +161,23 @@ fn wasmtime_config(ec: &EngineConfig) -> anyhow::Result<wasmtime::Config> {
     // wasmtime default: 4GB
     c.static_memory_maximum_size(instance_memory_maximum_size);
 
-    // wasmtime default: false
-    // We don't want threads, there is no way to ensure determisism
+    // wasmtime default: true
+    // We don't want threads, there is no way to ensure determinism
     c.wasm_threads(false);
 
     // wasmtime default: true
-    // simd isn't supported in wasm-instrument, but if we add support there, we can probably enable this.
+    // simd isn't supported in wasm-instrument, but if we add support there, we can probably enable
+    // this.
     // Note: stack limits may need adjusting after this is enabled
     c.wasm_simd(false);
```

Comment:
This is a bit sad; I haven't looked at wasm-instrument yet but this shouldn't be a big deal to enable should it? just lots of instructions and everything at 128 bits.